React/Views/RCTView.m

```diff
@@ -1667,6 +1667,12 @@ - (RCTViewKeyboardEvent*)keyboardEvent:(NSEvent*)event {
   NSArray<NSString *> *validKeys = keyDown ? self.validKeysDown : self.validKeysUp;
   NSString *key = [RCTViewKeyboardEvent keyFromEvent:event];
 
+  // If the view is focusable and the component didn't explicity set the validKeysDown or Up,
+  // allow enter/return and spacebar key events to mimic the behavior of native controls.
+  if (self.focusable && validKeys == nil) {
+    validKeys = @[@"Enter", @" "];
```

Comment:
Does this include both the Return and Enter keys on a Mac keyboard? They're technically different key codes.

Reply:
From `RCTViewKeyboardEvent.m`

```
  } else if (code == NSEnterCharacter || code == NSNewlineCharacter || code == NSCarriageReturnCharacter) {
    return @"Enter";
```

Reply:
I'm pretty sure the w3c web spec is that the `key` value of this event should say "Enter" : https://www.w3.org/TR/uievents-key/#keys-whitespace

That's what we follow for React Native Desktop.

Reply:
Cool, thanks!